prog.py

```diff
@@ -1,8 +1,13 @@
 def add(a, b):
 	return a + b
 
-def main():
-	print(add(1,2))
+def sub(a, b):
+	return a - b
 
+def main():
+	print("addition of two numbers: " + str(add(1,2)))
+	print("substraction of two numbers: " + str(sub(10,5)))
```

Comment:
please create seprate variables instead of hard coded values for add and sub methods